backend/user-service/src/controllers/auth.controller.ts

```diff
@@ -67,3 +75,91 @@ export async function hashPassword(password: string): Promise<string> {
     const saltRounds = 10
     return hash(password, saltRounds)
 }
+
+export async function getHTMLTemplate(htmlFilePath: string): Promise<string> {
+    const filePath = path.join(__dirname, htmlFilePath)
+    return await fs.readFile(filePath, 'utf8')
+}
+
+export async function sendMail(to: string, subject: string, text: string, html: string): Promise<void> {
+    const transporter = nodemailer.createTransport({
+        service: 'gmail',
+        host: 'smtp.gmail.com',
+        port: 587,
+        secure: false,
+        auth: {
+            user: process.env.EMAIL,
```

Comment:
you can add these environment variables into the config.util.ts file so that they can be verified on startup

Reply:
The names could perhaps be more descriptive, like NODEMAILER_EMAIL